src/partial-result-stream.js

```diff
@@ -96,6 +102,16 @@ function partialResultStream(requestFn, options) {
         formattedRows = formattedRows.concat(builder.toJSON());
         rowChunks.length = 0;
       } else {
+        // A streamed result set consists of a stream of values, which might
```

Comment:
You also need to handle the possibility of extra fields in the above if block since the last chunk in a list of chunks might have extra fields.